src/components/PositionsList/PositionItem/PositionItem.tsx

```diff
@@ -47,6 +49,33 @@ export const PositionItem: React.FC<IPositionItem> = ({
     initialXtoY(tickerToAddress(tokenXName), tickerToAddress(tokenYName))
   )
 
+  const getPercentageRatio = () => {
+    if (maxTick <= currentTick) {
+      return {
+        tokenXPercentage: xToY ? '0' : '100',
+        tokenYPercentage: xToY ? '100' : '0'
+      }
+    } else if (minTick >= currentTick) {
+      return {
+        tokenXPercentage: xToY ? '100' : '0',
+        tokenYPercentage: xToY ? '0' : '100'
+      }
+    } else {
+      const totalTicksRange = Math.abs(Number(maxTick - minTick))
+
+      const tokenXPercentage = xToY
```

Comment:
The result should be rounded up for the token that has a lower percentage and rounded down for the token that has a higher percentage. This will prevent a situation where the position is active but shows a ratio of 100% - 0%. It's better to round it to 99% - 1% because there is still some of the second token present

![Screenshot from 2024-08-08 05-17-27](https://github.com/user-attachments/assets/995b2aee-cf72-43bb-875b-a3426c11e70f)